modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -11,7 +11,8 @@ Amazon S3 Connector v5.5.0
 
 Guide: xref:connectors::amazon/amazon-s3-connector.adoc[Amazon S3 Connector]
 
-Release Notes: xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes]
+Release Notes: xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes] +
+Exchange: https://anypoint.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-s3-connector[Amazon S3 connector in Anypoint Exchange]
```

Comment:
Exchange link was intentionally removed. We don't want it at the top of the page.